crates/hc_deepkey_sdk/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "hc_deepkey_sdk"
+version = "0.6.0"
+authors = ["Matthew Brisebois <matthew.brisebois@holo.host>"]
+edition = "2018"
```

Comment:
I think this has been copied from an older version of `main` because this should be 2021 on `main`.

We potentially lose other changes if that's the case. Can this be recopied from the latest version of `main` please?